client/core/notification.go

```diff
@@ -84,11 +88,23 @@ func (c *Core) notify(n Notification) {
 // channel has capacity 1024, and should be monitored for the lifetime of the
 // Core. Blocking channels are silently ignored.
 func (c *Core) NotificationFeed() <-chan Notification {
```

Comment:
How 'bout we export this pattern for webserver, et. al?  Except it could be simplified:

```go
func (c *Core) NotificationFeed() (feed <-chan Notification, done func()) {
	ch := make(chan Notification, 1024)
	cid := atomic.AddUint64(&noteChanCounter, 1)
	c.noteMtx.Lock()
	c.noteChans[cid] = ch
	c.noteMtx.Unlock()
	return ch, func() {
		c.noteMtx.Lock()
		delete(c.noteChans, cid)
		c.noteMtx.Unlock()
	}
}
```